src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/Asn1/DistributionPointAsn.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<asn:Sequence
+  xmlns:asn="http://schemas.dot.net/asnxml/201808/"
+  name="DistributionPointAsn"
+  namespace="System.Security.Cryptography.X509Certificates.Asn1">
+
+  <!--
+    https://tools.ietf.org/html/rfc5280#section-4.2.1.13
+
+    DistributionPoint ::= SEQUENCE {
+        distributionPoint       [0]     DistributionPointName OPTIONAL,
+        reasons                 [1]     ReasonFlags OPTIONAL,
+        cRLIssuer               [2]     GeneralNames OPTIONAL
+    }
+  -->
+  <asn:AsnType name="DistributionPoint" typeName="System.Security.Cryptography.X509Certificates.Asn1.DistributionPointNameAsn" explicitTag="0" optional="true" />
```

Comment:
DistributionPoint is defined in ASN.1 module section A.2, which is `DEFINITIONS IMPLICIT TAGS`, so it seems like this one should be implicitTag, not explicitTag (like the other two)

Reply:
`DistributionPointNameAsn` is a choice and choices are always explicit.

Reply:
Ah, yes, that one :smile:.  Carry on.